src/utils.js

```diff
@@ -56,6 +56,18 @@ export const getContent = ({ mimeType, text }) => {
   return text;
 };
 
+export const sortHeaders = (current, next) => {
+  if (current.name < next.name) {
+    return -1;
+  }
+  return current.name > next.name ? 1 : 0;
+};
+
+export const getHeaders = (entry) => ({
+  request: entry.request.headers.sort(sortHeaders),
+  response: entry.response.headers.sort(sortHeaders),
```

Comment:
tests? 
also aren't headers.name a string? why do we need to sort them?

Reply:
To be consistent with chrome devtools header section, it's alphabetically sorted.